server/export.go

```diff
@@ -106,7 +109,18 @@ func ExportCmd(appExporter types.AppExporter, defaultNodeHome string) *cobra.Com
 
 			cmd.SetOut(cmd.OutOrStdout())
 			cmd.SetErr(cmd.OutOrStderr())
-			cmd.Println(string(sdk.MustSortJSON(encoded)))
+
+			out := sdk.MustSortJSON(encoded)
+			cmd.Println(string(out))
+			var exportedGenDoc tmtypes.GenesisDoc
+			if err = tmjson.Unmarshal(out, &exportedGenDoc); err != nil {
+				return err
+			}
+
+			if err = exportedGenDoc.SaveAs(outputPath); err != nil {
```

Comment:
What if the current directory isn't writable.
I think we want to as well keep the old behavior.